includes/Core/Authentication/Authentication.php

```diff
@@ -424,6 +424,33 @@ public function is_authenticated() {
 		return ! empty( $access_token );
 	}
 
+	/**
+	 * Checks whether the Site Kit setup is considered complete.
+	 *
+	 * If this is not the case, most permissions will be force-prevented to ensure that only permissions required for
+	 * initial setup are granted.
+	 *
+	 * @since 1.0.0
```

Comment:
This should be `@since n.e.x.t` now since the method is new _here_. Or you leave the `@since 1.0.0` and add a line below `@since n.e.x.t Moved from Permissions class.

Reply:
Ah, right on 👍